src/components/tag/tag.styles.ts

```diff
@@ -75,21 +75,21 @@ export default css`
 
   .tag--small {
     font-size: var(--sl-button-font-size-medium);
-    height: calc(var(--sl-input-height-small) * 0.85715);
+    height: calc(var(--sl-input-height-small) - 0.25rem); /* 24px */
```

Comment:
@CrookedGrin -- Here is that change from the `*` decimal to `-` a rem value! I don't know why github doesn't update the code block near the comment or at least give us the option of seeing the "old version" and the "new version"? 🤷‍♀️

Reply:
Hmm, weird. Anyway, looks good!